active-directory-new-domain-sql/README.md

```diff
@@ -0,0 +1,69 @@
+# Deploy a two Domain Controller VM and MS SQL VM.
+
+<a href= | https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ftcsatheesh%2Fazure-quickstart-templates%2Fmaster%2Factive-directory-new-domain-sql%2Fazuredeploy.json |  target= | _blank | >
+    <img src= | http://azuredeploy.net/deploybutton.png | />
+</a>
+
+Based on the https://github.com/Azure/azure-quickstart-templates/tree/master/windows-vm-custom-script and https://github.com/Azure/azure-quickstart-templates/tree/master/active-directory-new-domain-ha-2-dc
+
+This template allows you to deploy a Windows VM (default name adPDC) with a new Active Directory forest/domain. This is done by Desired State Configuration (CreateADPDC.ps1.zip).
+The second VM with SQL (default name adPDC) is configured powershell script (skrypt.ps1). The scripts must be in stored in a azure blob storage.
+
+You can connect to the AD by Remote Desktop (by your domainName.location.cloudapp.azure.com eg.: armadsqltst.westeurope.cloudapp.azure.com).
+Default username: adarmtest\adAdministrator

```

Comment:
It's a best practice to let users pick these explicitly rather than have these default. Also helps from security standpoint where a user forgets to change these and default username/password to their machine is out on the public internet.